tests/oracles/test_oracles.py

```diff
@@ -0,0 +1,40 @@
+from unittest import TestCase
+
+from pyrelational.oracles.benchmark_oracle import BenchmarkOracle
+from tests.test_utils import get_classification_dataset
+
+
+class TestOracle(TestCase):
```

Comment:
It's difficult to test as the benchmarkOracle won't change a thing in the datamanager. Since it only call other functions and is tested as the interface to the pipeline, I think it's fine to ignore it here.